service/telemetry/internal/migration/v0.2.0.go

```diff
@@ -37,13 +37,12 @@ type logsConfigV020 struct {
 }
 
 func headersV02ToV03(in configv02.Headers) []config.NameStringValuePair {
-	headers := make([]config.NameStringValuePair, len(in))
-	idx := 0
+	var headers []config.NameStringValuePair
```

Comment:
```suggestion
	headers := make([]config.NameStringValuePair, 0, len(in))
```

Reply:
TIL: append on a pre allocated slice doesn't immediately grow the slice